Automattic/wp-calypso
# Fix/remove paid feature for earn

## Proposed Changes

* Now that Earn features are on free plan,  Most of the gating/ nudges need to be removed

## Testing plan

* Run this diff with `yarn start`
* Go to any simple WPCOM site dashboard and make sure no nudge to upgrade a plan appears, but there are nudges for Stripe
